enhancements/installer/alibaba-cloud-ipi.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Alibaba Cloud IPI
+authors:
+  - "@lemondlut"
+reviewers:
+  - "@bradteng"
+  - "@rosawy"
+approvers:
+  - "@?"
+creation-date: 2021-03-25
+last-updated: 2021-03-25
+status: implementable
+
+---
+
+# Alibaba Cloud IPI
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement covers adding support to install OpenShift clusters to Alibaba Cloud and details for the Installer to create infrastructure and run an OpenShift cluster on Alibaba Cloud.
+
+## Motivation
+
+Currently, openshift4.x does not support Alibaba Cloud via IPI installation. However, the feedback team got is that there is a lot of demand for this kind of installation.
+
+Users interest in running Openshift in Alibaba Cloud can already do it manually. (Link TBD will be added after UPI document is published), but IPI installation will make it much simpler and automated.
+
+### Goals
+
+1. Installer creates infrastructure and installs OpenShift on Alibaba Cloud.
+2. Installer provides few but useful and comprehensive customizations through the install-config.yaml file.
+3. RHCOS images are built and published to Alibaba as part of the images pipeline process.
```

Comment:
I don't see much discussion of this goal in the document. What image pipeline process needs to be updated and what changes are needed?

Reply:
@ashcrow Could you or someone from your team update the information here regarding the RHCOS pipeline work? We would like to augment this section of the enhancement.

Reply:
cc @fabianofranz

Reply:
There is a document that was shared with Red Hat that elaborates the process.  Could we place the contents/ideas or process from that document into this enhancement repo?

Reply:
- Red Hat builds the RHCOS images from their build pipeline
- Red Hat will perform some validation
- RH will push RHCOS image to OSS package.
- RH will be required to push image to each region
- RH will update the installer with the correct image ID
Similar to this process https://github.com/openshift/installer/blob/master/data/data/rhcos-amd64.json#L2-L65